src/main.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App.tsx';
-import './index.css';
+import './styles/index.css';
```

Comment:
You already imported this in the index

Reply:
Can you review the PR please ?

Reply:
If a delete this line, the base style in index.css is not respected ... so I keep it in the file.

Reply:
hmmm that should not be the case, it mean we are missing the import process we shoudl'nt have duplicate style imports

Reply:
nice i saw you fixed it in [this commit](https://github.com/ipsa-scrypt/big-brain/commit/78b0694bcb116fc0d28323a18db040a1a9ef2094) 
All good for me!